tests/publiccloud/slem_basic.pm

```diff
@@ -30,12 +30,11 @@ sub check_avc {
 
     my $instance = $self->{my_instance};
     # Read the Access Vector Cache to check for SELinux denials
-    my $avc = $instance->ssh_script_output(cmd => 'sudo ausearch -ts boot -m avc --format raw | ( grep type=AVC || true )');
```

Comment:
Why remove the `grep`?

Reply:
We already specify `-m avc`

Reply:
There are also user-avc messages, that don't get filtered this way.

Reply:
so we will have MORE messages , I don't see big problem with that . @grisu48  I would suggest that we will try this way if it will get too spamy we may revert this line . Does it make sense for you ?

Reply:
Done. Please check again.